cmd/integrationArtifactGetMplStatus.go

```diff
@@ -119,3 +131,37 @@ func runIntegrationArtifactGetMplStatus(
 	log.Entry().Errorf("a HTTP error occurred! Response body: %v, Response status code: %v", responseBody, mplStatusResp.StatusCode)
 	return errors.Errorf("Unable to get integration flow MPL status, Response Status code: %v", mplStatusResp.StatusCode)
 }
+
+//getIntegrationArtifactMPLError - Get integration artifact MPL error details
+func getIntegrationArtifactMPLError(commonPipelineEnvironment *integrationArtifactGetMplStatusCommonPipelineEnvironment, mplId string, httpClient piperhttp.Sender, apiHost string) (string, error) {
```

Comment:
Also here it we could switch to mplID rather than mplId

Reply:
done